debian/changelog

```diff
@@ -1,3 +1,27 @@
+cloud-init (20.2-45-g5f7825e2-0ubuntu1~19.10.1) eoan; urgency=medium
+
+  * d/cloud-init.templates: add RbxCloud data source
+  * d/control: drop python3-six, python3-pyflakes and python3-pep8 from
+    Build-Depends
+  * d/patches/ec2-dont-apply-full-imds-network-config.patch: by default only
+    configure DHCP on eth0 to retain original netwwork config behavior
+  * d/patches/renderer-do-not-prefer-netplan.patch: leave default renderer
+    order as ENI before netplan.
+  * refresh patches:
```

Comment:
These patches are new in this version, you shouldn't have needed to refresh them.

Reply:
this was a result of me new-upstream-snapshotting on top of this WIP branch. restarting fresh, this is cleared up now